socorro/lib/util.py

```diff
@@ -40,12 +41,15 @@ def drop_unicode(text):
     :returns: text with all unicode characters dropped
 
     """
-    if isinstance(text, str):
-        # Convert any str to a unicode so that we can convert it back and drop any non-ascii
-        # characters
-        text = text.decode('unicode_escape')
-
-    return text.encode('ascii', 'ignore')
+    if sys.version_info < (3,):
```

Comment:
`six` has a `PY2` and `PY3` which are true and false. I'd use that instead since it's more readable in intent (in my opinion) and it's easy to grep for later. So then this line would be:

```
if six.PY2:
```